README.md

```diff
@@ -390,17 +428,20 @@ This tells you that you've successfully staged `user_info/Pat_and_Chris.md`.
 It also tells you how you can _unstage_ a file that you might have staged by
 accident.
 
-**Commit the change** _to your local copy of the repository_ with `git commit`
-on the command line.
+**Commit the change** _to your local copy of the repository_ with
 
-- This will bring up a (command line) editor for you to enter
+```bash
+git commit
+```
+
+* This will bring up a (command line) editor for you to enter
```

Comment:
The editor may not be a command line one. For students on Windows we have even suggested they change the setting when installing Git to VS Code:

![image](https://user-images.githubusercontent.com/1300395/104276747-1d9d1600-546b-11eb-98f3-0a45777ac4d7.png)

(https://gist.github.com/floogulinc/2d9f80b278332b0df731e777c43b206c#install-git)


Though vim is the default when you run through the installer and we could update the setup guide to suggest this. 

![image](https://user-images.githubusercontent.com/1300395/104276941-808ead00-546b-11eb-9c0f-eca69115fc16.png)

Reply:
Thanks for that. I'm fine with them using VS Code as their editor, so I'll just update my text.